FogCreek/Glitch-Community
# Smart fridge: add LuckyOrange tracking script

## Links
* Remix link: https://smart-fridge.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3328768/LuckyOrange-integration

## Changes:
* add LuckyOrange tracking script

## How To Test:
* visit https://smart-fridge.glitch.me/
* make sure your adblocker is turned off
* verify `window._loq` is available in the console
